src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -72,16 +72,24 @@ public function getLocator()
      */
     public function import($resource, ?string $type = null, bool $ignoreErrors = false, ?string $sourceResource = null, $exclude = null)
     {
-        if (\is_string($resource) && \strlen($resource) !== ($i = strcspn($resource, '*?{[')) && !str_contains($resource, "\n")) {
-            $excluded = [];
-            foreach ((array) $exclude as $pattern) {
-                foreach ($this->glob($pattern, true, $_, false, true) as $path => $info) {
-                    // normalize Windows slashes and remove trailing slashes
-                    $excluded[rtrim(str_replace('\\', '/', $path), '/')] = true;
-                }
+        $excluded = [];
+        $isGlobPattern = \strlen($resource) !== strcspn($resource, '*?{[');
```

Comment:
Will throw if $resource is not a string

Reply:
> Will throw if $resource is not a string

Yes, something was missing here... I added a condition to apply the exclude only if `resource` is a string, and is not an existing class FQCN.